test/e2e/wmco_test.go

```diff
@@ -31,10 +31,15 @@ func TestWMCO(t *testing.T) {
 		wmcoPath = "/usr/local/bin/windows-machine-config-operator"
 	}
 
+	testCtx, err := NewTestContext()
+	require.NoError(t, err)
+	require.NoError(t, testCtx.ensureNamespace(testCtx.workloadNamespace), "error creating test namespace")
```

Comment:
Doesnt this need to be cleaned up?

Reply:
That [happens](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/delete_test.go#L69) in the deletion tests